jerry-core/parser/regexp/re-parser.h

```diff
@@ -78,7 +78,7 @@ typedef enum
 /**
  * Undefined character (out of the range of the codeunit)
  */
-#define RE_CHAR_UNDEF 0xFFFFFFFF
+#define RE_CHAR_UNDEF 0xFFFF
```

Comment:
Is this correct? The code unit type seems to cover the whole 16 bits. This change would make the 16bit 0xFFFF a special invalid value. The original code had a 32 bit value that was indeed out of the range of the code unit type, as the doc states.